### ISSUE TITLE
Feature request: move and link card

### ISSUE THREAD

**Issue by superlou** [wekan/wekan#2719]
Is it possible to add an option to the "move card" menu to move the card and create a link on the original board?

My current workflow has a personal board for tasks assigned to me, and project boards that include all cards for a project. I often create new tasks on this personal board, and then move them to the project-specific board where other team members can see them. I then link the new card back into my personal board. If there was a checkbox in the move options to "create link" after the move, that would save a step.

**Comment by xet7** [comment]
When there is some important card at some other board, go to your `Overview of important tasks board` and click `Add Card` and there select `Link` so you can link to card that is at other board. Then both cards have same content (although, there could be bugs, I did not check).

**Comment by xet7** [comment]
Hmm, I'll re-read your question, I'm not sure yet did I get it right...

**Comment by xet7** [comment]
So that option would be for example:
- Move and link to
- Move as linked

So card then gets moved, and to it's same place is created linked card that points to moved card.

**Comment by xet7** [comment]
Actually, instead there at Move popup there could be checkbox:
```
[_] Replace current card with link to moved card
```
Maybe that would be more easier to understand?

**Comment by superlou** [comment]
Maybe I have an XY problem. I didn't know about the important card feature, though I haven't found it yet in the UI. We're currently running 3.44.0. Is it a recent addition? The root cause for my workflow is that I don't have an easy way to "view all cards I'm responsible for." I'm using the manual "personal board" in lieu of something like https://github.com/wekan/wekan/issues/1667.

I think the last suggestion of the checkbox is closest to what I was looking for. It's almost like I'm trying to add a new card quickly to another board and link it here without losing my context and switching to that board.

**Comment by xet7** [comment]
There is plans for that "My Cards" feature at https://github.com/wekan/wekan/issues/641#issuecomment-468868580 but it is not implemented yet.

**Comment by xet7** [comment]
Yes, I think this Move+Link is still too many steps. My Cards feature would be better.

**Comment by xet7** [comment]
Currently, there is too many steps to create the same effect:

1) Create new board called `My Cards` or `Overview of important tasks`

2) Go to board `My Cards`

3) Add new Link to card, that links to another card on another board

4) Then you see all important cards at `My Cards` board.

**Comment by xet7** [comment]
Link to card was added about one year ago https://github.com/wekan/wekan/pull/1592

**Comment by superlou** [comment]
Agreed that it is kind of a lot of steps. The "move and create link" compromise seemed like a way to handle most of my own workflow issues with a single checkbox.

There is actually a small advantage to the "move and create link" solution in that a personal board can organize the linked cards however it wants in lists, while a "view all my cards" option may be hard to condense since each board could use different list names.

**Comment by xet7** [comment]
Sorry, Wekan Features page is old https://github.com/wekan/wekan/wiki/Features

There are some other features listed at wiki, but many features are still only at ChangeLog:
https://github.com/wekan/wekan/blob/master/CHANGELOG.md

There is issue where I am asking help with documentation #2709

**Comment by xet7** [comment]
> There is actually a small advantage to the "move and create link" solution in that a personal board can organize the linked cards however it wants in lists, while a "view all my cards" option may be hard to condense since each board could use different list names.

Yes, being able to organize is an advantage, there is also discussion that organizing boards at `All Boards` page would be nice: https://github.com/wekan/wekan/issues/641#issuecomment-520105769
